### DISCUSSION TITLE
is .js pattern in example useful?

### DISCUSSION THREAD

**Discussion by mightyiam** [standard/eslint-config-standard-with-typescript#1977] [General]
I'd love some user input on this. Are you applying this to JS files, as well as TS files? Is that practical?

**Comment by gtbuchanan** [comment]
I am applying it to JS config files if the tool, such as ESLint, does not support TypeScript (e.g. `eslint.config.mjs`).

**Comment by vladpuz** [comment]
To make the configuration work with js, I do this https://github.com/mightyiam/eslint-config-love/issues/1525#issuecomment-2753725830
